dot/sync/disjoint_block_set_test.go

```diff
@@ -0,0 +1,313 @@
+// Copyright 2021 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package sync
+
+import (
+	"errors"
+	"testing"
+	"time"
+
+	"github.com/ChainSafe/gossamer/dot/types"
+	"github.com/ChainSafe/gossamer/lib/common"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_disjointBlockSet_addBlock(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
+		name             string
+		disjointBlockSet disjointBlockSet
+		block            *types.Block
+		err              error
+	}{
+		{
+			name: "add block beyond capacity",
+			block: &types.Block{
+				Header: types.Header{
+					Number: 1,
+				},
+			},
+			err: errors.New("cannot add block; set is at capacity"),
```

Comment:
nit use the corresponding sentinel (global scope) error here; if it doesn't exist please define it in the production code

Reply:
Updated to use sentinel error.